bgpd/bgp_nht.c

```diff
@@ -707,13 +707,16 @@ static void evaluate_paths(struct bgp_nexthop_cache *bnc)
 	if (BGP_DEBUG(nht, NHT)) {
 		char buf[PREFIX2STR_BUFFER];
 		char bnc_buf[BNC_FLAG_DUMP_SIZE];
+		char chg_buf[BNC_FLAG_DUMP_SIZE];
 
 		bnc_str(bnc, buf, PREFIX2STR_BUFFER);
 		zlog_debug(
-			"NH update for %s(%u)(%s) - flags %s chgflags 0x%x - evaluate paths",
+			"NH update for %s(%u)(%s) - flags %s chgflags %s - evaluate paths",
```

Comment:
tiny nit - you can probably remove the space before the '-' here right?

Reply:
fixed